runtime/gc_modron_startup/mmparseXgcpolicy.cpp

```diff
@@ -82,6 +83,15 @@ isSubpoolAliasGCPolicySupported(MM_GCExtensions *extensions)
 bool MMINLINE
 isMetronomeGCPolicySupported(MM_GCExtensions *extensions)
 {
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	J9JavaVM *vm = extensions->getJavaVM();
+	if (TRUE == vm->checkpointState.isCheckPointEnabled) {
```

Comment:
This should probably use the CRIU isEnabled API...something like `vm->internalVMFunctions->isCRIUSupportEnabled(vm->internalVMFunctions->currentVMThread(vm))` 

However, current thread is NULL and resulting in segfault that this point in startup.